multicluster/build/yamls/token-secret.yml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
```

Comment:
`build/yamls` folder is for general menifests,  you should place any specific files for integration test to its own folder.